README.md

```diff
@@ -84,25 +65,23 @@ const options = {
     ciphers: 'DES-CBC3-SHA' // The TLS ciphers to support
 };
 
-const optionalCallback = (err, res) => {
-
-    /* handle err if it exists, in which case res will be undefined */
-
-    // buffer the response stream
-    Wreck.read(res, null, (err, body) => {
-        /* do stuff */
-    });
-};
-
-const req = wreck.request(method, uri, options, optionalCallback);
+const promise = wreck.request(method, uri, options);
+// Use promise.req.abort() to terminate the request early
```

Comment:
This is not that simple. `promise.req` will be the wrong one, if the request has been redirected.

Reply:
How is this different from previous code?

Reply:
Aborting is useful for some use-cases. Anyway, it's primarily the comment I have an issue with, as it implies that aborting will just work like that.

Reply:
Fair.